apps/api/src/app/notification-template/usecases/create-notification-template/create-notification-template.usecase.ts

```diff
@@ -68,6 +68,7 @@ export class CreateNotificationTemplate {
           title: message.template.title,
           feedId: message.template.feedId,
           parentChangeId,
+          avatarDetails: { ...message.template.avatarDetails },
```

Comment:
is creating a new object required here? I see that for ex. `cta` field is passed as ref... 
and in the `update-notification-template.usecase.ts` you don't do the same...

Reply:
I agree, we don't need the spread operator here, might have left it out during refactoring